src/reducers/cacheReducer.js

```diff
@@ -150,7 +150,16 @@ const fieldsTransducer = (fields) =>
 const orderTransducer = (order) => {
   const isFlat = typeof order[0] === 'string';
   const orders = isFlat ? [order] : order;
-  return partialRight.apply(null, [orderBy, ...zip.apply(null, orders)]);
+  const [fields, direction] = zip(
+    ...orders.map(([field, dir]) => [
+      (data) =>
+        typeof data[field] === 'string'
+          ? data[field].toLowerCase()
+          : data[field],
+      dir || 'asc',
+    ]),
+  );
+  return partialRight(map, (docs) => orderBy(docs, fields, direction));
```

Comment:
I found a difference between Firestore's ordering and lo lodash ordering. So this fixes that.